packages/components/Tabs/src/TabsItem.tsx

```diff
@@ -0,0 +1,159 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View } from 'react-native';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { Text } from '@fluentui-react-native/text';
+import { Icon } from '@fluentui-react-native/icon';
+import { settings, tabsItemSelectActionLabel } from './TabsItem.settings';
+import { backgroundColorTokens, borderTokens, textTokens, foregroundColorTokens, getPaletteFromTheme } from '@fluentui-react-native/tokens';
+import { filterViewProps } from '@fluentui-react-native/adapters';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { TabsContext } from './Tabs';
+import { tabsItemName, TabsItemType, TabsItemProps, TabsItemSlotProps, TabsItemRenderData, TabsItemState } from './TabsItem.types';
+import {
+  useAsPressable,
+  useKeyCallback,
+  useViewCommandFocus,
+  createIconProps,
+  useOnPressWithFocus,
+} from '@fluentui-react-native/interactive-hooks';
+
+export const TabsItem = compose<TabsItemType>({
+  displayName: tabsItemName,
+
+  usePrepareProps: (userProps: TabsItemProps, useStyling: IUseComposeStyling<TabsItemType>) => {
+    const {
+      icon,
+      headerText = '',
+      onAccessibilityTap = userProps.onClick,
+      accessibilityLabel = userProps.headerText,
+      componentRef = React.useRef(null),
```

Comment:
Same comment I made in Tabs.tsx. Refer to https://github.com/microsoft/fluentui-react-native/pull/841